libcodechecker/analyze/analyzers/analyzer_clangsa.py

```diff
@@ -128,15 +128,12 @@ def construct_analyzer_cmd(self, result_handler):
                                          '-Xclang', checker_name])
 
             if config.ctu_dir:
-                env = analyzer_env.get_check_env(config.path_env_extra,
-                                                 config.ld_lib_path_extra)
-                triple_arch = ctu_triple_arch.get_triple_arch(self.buildaction,
-                                                              self.source_file,
-                                                              config, env)
                 analyzer_cmd.extend(['-Xclang', '-analyzer-config',
                                      '-Xclang',
-                                     'xtu-dir=' + os.path.join(config.ctu_dir,
-                                                               triple_arch)])
+                                     'xtu-dir=' + self.get_xtu_dir(),
+                                     '-Xclang',
+                                     '-analyzer-display-ctu-progress',
```

Comment:
Good point. The analysis will fail if the used Clang does not support this switch.
This will cause a problem for CodeChecker in the time window when the CTU patch is accepted but the patch for this switch is still not.
I hope this time window will be short, since the patch for `-analyzer-display-ctu-progress` is very tiny. Still this is a valid danger.
We have the following options:
* Create a framework to discover feature capabilities of the used Clang (this is a huge work, counting in men weeks I estimate)
* If the command fails, then retry without `-analyzer-display-ctu-progress` switch
* Any other ideas?